bootstrap/service_test.go

```diff
@@ -95,6 +96,17 @@ func newThingsServer(csvc clients.Service, gsvc groups.Service, psvc tpolicies.S
 	capi.MakeHandler(csvc, mux, logger)
 	gapi.MakeHandler(gsvc, mux, logger)
 	papi.MakeHandler(csvc, psvc, mux, logger)
+
+	// Define a custom route to redirect /things/policies to /policies
+	mux.HandleFunc("/things/policies", func(w http.ResponseWriter, req *http.Request) {
+		http.Redirect(w, req, "/policies", http.StatusPermanentRedirect)
+	})
+
+	// Define a custom route to redirect /things/policies/sub/obj to /policies/sub/obj
+	mux.HandleFunc("/things/policies/:sub/:obj", func(w http.ResponseWriter, req *http.Request) {
+		http.Redirect(w, req, fmt.Sprintf("/policies/%s/%s", bone.GetValue(req, "sub"), bone.GetValue(req, "obj")), http.StatusPermanentRedirect)
+	})
```

Comment:
Same here, is this necessary? One can use the port to distinct users from things locally.

Reply:
The SDK assumes we are in front a a proxy hence we don't use ports to communicate to `things` and `users`